samples/client/petstore/k6/script.js

```diff
@@ -161,8 +161,8 @@ export default function() {
         sleep(SLEEP_DURATION);
     });
     group("/user/login", () => {
-        let password = "TODO_EDIT_THE_PASSWORD";
-        let username = "TODO_EDIT_THE_USERNAME";
+		let password = 'TODO_EDIT_THE_PASSWORD'; // specify value as there is no example value for this parameter in OpenAPI spec
+		let username = 'TODO_EDIT_THE_USERNAME'; // specify value as there is no example value for this parameter in OpenAPI spec
```

Comment:
Correct me if I'm wrong, but I expect these values to be filled with examples? Or is it another example script altogether? If so, please regenerate that example alongside this one.

Reply:
Hi 

Thank you for the feedback. My IDE does not understand mustache (yet), so apologies for the indentation problems; I’ll fix it ASAP.

Regarding the `samples/client/petstore/k6/script.js`, please keep in mind samples are generated based on whatever is defined in the config, and for K6 generator, the config file `https://github.com/OpenAPITools/openapi-generator/blob/master/bin/configs/other/k6.yaml` uses `modules/openapi-generator/src/test/resources/2_0/petstore.yaml` as the input spec; unfortunately, that OpenAPI spec (`modules/openapi-generator/src/test/resources/2_0/petstore.yaml`) does not have any examples, and since it’s used by many other generators I did not want to introduce any changes in such place; this is also the reason why in the docs for this PR I provided example (“Generate script based on Swagger/OpenAPI specification with examples”) based on `modules/openapi-generator/src/test/resources/3_0/examples.yaml`. 

Whether `modules/openapi-generator/src/test/resources/2_0/petstore.yaml` itself should be updated with examples – perhaps that’s a good question to ask the OpenAPI maintainers?

Reply:
@wing328 Any comments?

Reply:
> modules/openapi-generator/src/test/resources/2_0/petstore.yaml

We usually keep it as it's and create a new one based on it with more edge cases per generator.